src/App.jsx

```diff
@@ -1,5 +1,7 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import cn from 'classnames';
```

Comment:
Ensure that the `classnames` package is installed in your project. You can install it using `npm install classnames` if it's not already installed.